pack.yaml

```
@@ -6,7 +6,7 @@ keywords:
   - LogicMonitor
   - monitoring
   - alerting
-version: 1.1.0
+version: 1.2.0
```

Since this PR is not changing anything major and it's just a bugfix, bumping a patch version would be more appropriate

```suggestion
version: 1.1.1
```

See https://semver.org/#spec-item-6